BehaviorTreeEditor.cs

```diff
@@ -44,24 +61,120 @@ public static void ShowWindow()
 
 	void OnGUI()
 	{
+		try
+		{
+			CreateGUI();
+			AssetDatabase.SaveAssets();
+		}
+		catch ( Exception e )
+		{
+			Debug.LogError( e );
```

Comment:
I hate it so much that this is necessary. It makes debugging suck because you can't click on the error to go to the line that caused it, but if I don't have this it locks up the editor.